nifi/Dockerfile

```diff
@@ -4,6 +4,65 @@
 # Ignoring DL4006 globally because we inherit the SHELL from our base image
 # hadolint global ignore=DL3038,DL4006
 
+FROM stackable/image/java-devel AS builder
+
+ARG PRODUCT
+
+RUN microdnf update && \
+    microdnf clean all && \
+    rm -rf /var/cache/yum
+
+COPY nifi/licenses /licenses
```

Comment:
Are these used later? You copy them again in the final image.

Reply:
No its a left over, removed https://github.com/stackabletech/docker-images/pull/678/commits/e623a6a66e37f55119045aa5001309e885cc14db